inst/include/forcing_component.hpp

```diff
@@ -120,7 +120,7 @@ class ForcingComponent : public IModelComponent {
 
   // Aerosol parameters for aerosol-cloud interactions (RFaci) see equation
   // Equation 7.SM.1.2 of IPCC AR6
-  double const aci_beta = 2.09841432; // (W m-2) IPCC AR6 7.SM.1.3.1
+  double const aci_beta = 2.279759; // see Dorheim et al. in prep
```

Comment:
Add a TODO or a GitHub issue so we remember to come back and update this

Reply:
Up there are a whole bunch of Dorheim et al. in prep citations after publication

Reply:
See issue #725